pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -813,6 +817,32 @@ cdef class _Timedelta(timedelta):
         else:
             return "D"
 
+    @property
+    def resolution_timedelta(self):
+        """
+        Return a timedelta object (rather than a string)
+        representing the lowest resolution we have.
```

Comment:
Instead of `lowest resolution that we have`, I think I'd be clearer if it was `lowest resolution of the Timedelta class`